.github/workflows/build-push-docker.yaml

```diff
@@ -93,7 +93,7 @@ jobs:
             "ghcr.io/${{ github.repository_owner }}/nebari-${{ matrix.dockerfile }}${{ env.IMAGE_SUFFIX }}"
           tags: |
             # branch event -> e.g. `main-f0f6994-20221001`
-            type=ref,event=branch, suffix={{sha}}-{{date 'YYYYMMDD'}}
+            type=ref,event=branch, suffix=-{{sha}}-{{date 'YYYYMMDD'}}
```

Comment:
We have been using the `main` tag for testing in other repos (kubernetes tests, integration tests, etc) but the addition of this suffix will break those. If we want both, can we also add a tag for the branch name?

Reply:
@trallard

Reply:
well - last time we discussed the tagging format for the Docker images this is what we agreed on (also why I was asking where all the Docker images needed updating)
either we add another tag here or fix elsewhere

Reply:
@trallard in my defense, I recall having this conversation and I also recall pushing back for the reason stated above. I just realized that this change will also break how we initialize the default image tags in the `nebari-config.yaml::default_images` section.